<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myindex</class>
 <widget class="QMainWindow" name="myindex">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>员工考勤系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>871</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>871</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_add">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>456</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>添加员工信息</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_delete">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>456</y>
        <width>96</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>删除员工信息</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_update">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>456</y>
        <width>96</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>修改员工信息</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_find">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>456</y>
        <width>96</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查询员工信息</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_refresh">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>456</y>
        <width>96</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>全部员工信息</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_exit">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>456</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>退出系统</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableView" name="tableView_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_back">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>470</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>返回主界面</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_find_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>470</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_all">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>470</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>全部</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>421</width>
        <height>491</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QWidget" name="widget_find" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>391</width>
          <height>491</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_cardID">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>140</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>卡号:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_name">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>170</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>姓名:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_phone">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>210</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>电话:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_ID">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>250</y>
           <width>64</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>身份证号:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_cardID">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>130</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_name">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>170</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_department">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>290</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>董事会</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>销售部</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>技术部</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>后勤部</string>
          </property>
         </item>
        </widget>
        <widget class="QComboBox" name="comboBox_position">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>330</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>董事长</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>总经理</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>经理</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>副经理</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>员工</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_department">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>300</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>部门:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_position">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>330</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>职位:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_phone">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>210</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_ID">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>250</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_starttime">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>10</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>起始日期</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_endtime">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>50</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>终止日期</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_cancle">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>440</y>
           <width>71</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_clear">
         <property name="geometry">
          <rect>
           <x>175</x>
           <y>440</y>
           <width>71</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_OK">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>440</y>
           <width>71</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>96</y>
           <width>341</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>请选择输入下面的任意几项:</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit_start">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>10</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit_end">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>50</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QCalendarWidget" name="calendarWidget_start">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>10</y>
           <width>281</width>
           <height>172</height>
          </rect>
         </property>
        </widget>
        <widget class="QCalendarWidget" name="calendarWidget_end">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>60</y>
           <width>281</width>
           <height>172</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QWidget" name="widget_look" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>391</width>
          <height>491</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>260</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>卡号:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>10</y>
           <width>141</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>原始照片</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>10</y>
           <width>131</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>刷卡时采集到的照片</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_org_photo">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>40</y>
           <width>141</width>
           <height>181</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_cur_photo">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>40</y>
           <width>141</width>
           <height>181</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>260</y>
           <width>31</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>姓名:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>310</y>
           <width>31</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>电话:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>310</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>性别:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>360</y>
           <width>31</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>部门:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>360</y>
           <width>31</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>职位:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_cardID_2">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>260</y>
           <width>91</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_name_2">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>260</y>
           <width>91</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_sex_2">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>310</y>
           <width>81</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_phone_2">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>310</y>
           <width>101</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_department_2">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>360</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_position_2">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>360</y>
           <width>101</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>440</y>
           <width>96</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>  基本功能  </string>
    </property>
    <addaction name="action_staff_manage"/>
    <addaction name="separator"/>
    <addaction name="action_workinfo"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>  帮助  </string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_staff_manage">
   <property name="text">
    <string>员工基本信息管理</string>
   </property>
  </action>
  <action name="action_workinfo">
   <property name="text">
    <string>员工考勤信息查询</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
